### PR TITLE
Docker image for ppc64le arch

### PR DESCRIPTION
**Pull request by mkumatag**
This PR is for supporting docker image for ppc64le platform.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -59,11 +61,15 @@ check-docker:
 docker-binary: BINDIR = ./rootfs
 docker-binary: GOFLAGS += -a -installsuffix cgo
 docker-binary:
-	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 $(GO) build -o $(BINDIR)/tiller $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' k8s.io/helm/cmd/tiller
+	GOOS=linux GOARCH=$(ARCH) CGO_ENABLED=0 $(GO) build -o $(BINDIR)/tiller $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' k8s.io/helm/cmd/tiller
```
Comment (adamreese) [inline Makefile]: Should GOARCH always be host arch when building for docker?

Comment (mkumatag) [inline Makefile]: Yes.. as docker cross building is not supported, we have to be on target host arch to build docker image..

**Comment by technosophos** [comment]
This and #1930 are raising the concern that maybe we need to revisit how the entire canary/release build process is working. The backpressure caused in CI by increasing the build time is having a negative effect on handling regular pull requests (since the build queue keeps getting longer and longer).

If we can offload some of the build pressure by making a CI/CD change then we can add things like dockerized builds for lots of platforms, and do so without increasing the build wait time to :45.

So let's hold off on merging this and #1930 until after 2.2.0 (so nobody gets distracted in these last few days), and then we can tackle the CI/CD situation without the risk of stalling the Feb. 14 release date.

**Comment by thomastaylor312** [comment]
@technosophos Just circling back to this. Maybe we should talk about the CI/CD stuff in one of the next public standup meetings

**Comment by thomastaylor312** [comment]
Bumping this back to Upcoming - Minor. @technosophos we should have a conversation about whether or not this is possible now

**Comment by thomastaylor312** [comment]
Side note, @technosophos we really do need to discuss again if this would be ok to do now. Maybe @adamreese has some thoughts too

**Comment by bacongobbler** [comment]
Let's bring this up in the next dev call.

**Review by bacongobbler — COMMENTED** [review]
What kubernetes clusters run on ppc64le architectures? I don't even think kubernetes itself supports running on ppc64le at this point. https://github.com/kubernetes/kubernetes/issues/38067

To be clear, I'm all for enabling all architecture types. I just want to be cautious about adding support for running Helm on alternative chip architectures that kubernetes core is unprepared to support.

**Comment by bacongobbler** [comment]
@mkumatag I see you're working on this upstream. Any idea on a general timeline when upstream should be able to support ppc64le architectures? :)

**Comment by mkumatag** [comment]
> What kubernetes clusters run on ppc64le architectures? I don't even think kubernetes itself supports running on ppc64le at this point. kubernetes/kubernetes#38067
> 
> To be clear, I'm all for enabling all architecture types. I just want to be cautious about adding support for running Helm on alternative chip architectures that kubernetes core is unprepared to support.

kubernetes/kubernetes#38067 issue is ubmrella issue which is pointing to lot of thing. Currently kubernetes is building binaries for most of the architectures, you can find bits @ https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.11.md#v1110-alpha1. And even we run tests on ppc64le platform and published in [test-grid](https://k8s-testgrid.appspot.com/sig-node-ppc64le#Summary&show-stale-tests=).

**Review by bacongobbler — CHANGES_REQUESTED** [review]
It sounds like @alicefr is interested in working on making it possible to add support for various architectures, not just arm64 or ppc64le. It might be good to get in contact with her and see if she's interested in collaborating. :)

**Comment by alicefr** [comment]
hi everybody! Please see my [answer](https://github.com/helm/helm/pull/4252#issuecomment-411166537). I'm interested in participating to the discussion!

**Comment by bacongobbler** [comment]
closing as irrelevant for Helm 3 as Tiller is being removed, and therefore there's no need to build a ppc64le docker image. Thanks!

**Pull request closed without merging**